egs/wsj0-mix/DeepClustering/local/conf.yml

```diff
@@ -18,13 +22,18 @@ optim:
   weight_decay: 0.
 # Data config
 data:
-  tr_wav_len_list: /srv/storage/talc3@talc-data.nancy/multispeech/calcul/users/ssivasankaran/experiments/data/speech_separation/wsj0-mix/2speakers_reverb_kinect_chime_noise_corrected/wav16k/max/tr.wavid.sample.10
-  cv_wav_len_list: /srv/storage/talc3@talc-data.nancy/multispeech/calcul/users/ssivasankaran/experiments/data/speech_separation/wsj0-mix/2speakers_reverb_kinect_chime_noise_corrected/wav16k/max/cv.wavid.sample.10
-  wav_base_path: /srv/storage/talc3@talc-data.nancy/multispeech/calcul/users/ssivasankaran/experiments/data/speech_separation/wsj0-mix/2speakers_reverb_kinect_chime_noise_corrected/wav16k/max/
+    #  tr_wav_len_list: /srv/storage/talc3@talc-data.nancy/multispeech/calcul/users/ssivasankaran/experiments/data/speech_separation/wsj0-mix/2speakers_reverb_kinect_chime_noise_corrected/wav16k/max/tr.wavid.sample
+    #  cv_wav_len_list: /srv/storage/talc3@talc-data.nancy/multispeech/calcul/users/ssivasankaran/experiments/data/speech_separation/wsj0-mix/2speakers_reverb_kinect_chime_noise_corrected/wav16k/max/cv.wavid.sample
+    #  tt_wav_len_list: /srv/storage/talc3@talc-data.nancy/multispeech/calcul/users/ssivasankaran/experiments/data/speech_separation/wsj0-mix/2speakers_reverb_kinect_chime_noise_corrected/wav16k/max/tt.wavid.sample.head
+  tr_wav_len_list: exp/tr.wavid.samples
+  cv_wav_len_list: exp/cv.wavid.samples
+  tt_wav_len_list: exp/tt.wavid.samples
+  wav_base_path: /srv/storage/talc3@talc-data.nancy/multispeech/calcul/users/ssivasankaran/experiments/data/speech_separation/wsj0-mix/2speakers/wav8k/min/
```

Comment:
The config file shouldn't need modification, absolute path have to be in the run.sh. 
The text files containing infos about the dataset have to be under `./data/`